src/plugins/intel_gna/src/transformations/utils/gather_sinking_utils.cpp

```diff
@@ -49,77 +51,167 @@ GatherInputsInfo GetFirstGatherInput(NodePtr node) {
     return GatherInputsInfo();
 }
 
-bool IfNodeHasGatherInputs(const Output<Node>& output) {
-    GatherInputsInfo inputs_info = GetFirstGatherInput(output.get_node_shared_ptr());
-    return !inputs_info.isEmpty();
+void swap_output_names(Output<Node> output1, Output<Node> output2) {
```

Comment:
this function looks common for all transformations, please move it to graph_utils